scripts/generate_markdowns.py

```diff
@@ -36,8 +36,8 @@ def generate_markdown_from_notebook(config, tutorial, output_path, tutorials_pat
     md_exporter = MarkdownExporter(exclude_output=True)
     body, _ = md_exporter.from_filename(f"{tutorials_path}")
     print(f"Processing {tutorials_path}")
-
-    with open(f"{output_path}/{tutorial['notebook'][:-6]}.md", "w", encoding="utf-8") as f:
+    filename = f"{tutorial['notebook'][:-6]}" if 'slug' not in tutorial else tutorial['slug']
```

Comment:
nit: you can use a shorter form
`filename = tutorial.get('slug', tutorial['notebook'][:-6])`

Reply:
Oh cool yes, so this is 'get value of 'key' if it exists, if not apply second parameter (value) correct?